content/components/data-table.mdx

```diff
@@ -0,0 +1,604 @@
+/>
+</Box>
+
+### Filtering
+
+<Box display="flex" flexDirection={['column', 'column', 'column', 'column', 'row']} sx={{gap: 3}}>
+
+<Box as="p" flex={1}>
+  The "query builder" pattern is used to show rows that match the user's query.
+</Box>
+
+<img
+  width="456"
+  role="presentation"
+  src="https://user-images.githubusercontent.com/2313998/205361241-318146a6-d2ff-47e7-96fc-4f1bd4cd5698.png"
+/>
+</Box>
+
+## Accessibility
+
+### Labeling and describing the table
+
+The table must have a title, and can optionally have a subtitle to describe more context. If you encounter a case where there is enough context that the table doesn’t need a visible label, you may hide it so it’s only accessible to assistive technologies.
+
+<Box display="flex" flexDirection={['column', 'column', 'column', 'column', 'row']} mb={3} sx={{gap: 3}}>
+
+<Box as="p" flex={1}>
+  If you use the table header, the table's title and subtitle will automatically be associated with the table for
+  assistive technologies.
+</Box>
+
+<img
+  width="456"
+  role="presentation"
+  src="https://user-images.githubusercontent.com/2313998/205361243-62d0b497-a4ba-44b5-8e3e-258cdd6e53e1.png"
+/>
+</Box>
+
+<Box display="flex" flexDirection={['column', 'column', 'column', 'column', 'row']} sx={{gap: 3}}>
+
+<Box as="p" flex={1}>
+  If your table's title exists somewhere else on the page (for example, in the PageHeader), then you must manually
+  associate that contextual information with the table using ARIA.
+</Box>
+
+<img
+  width="456"
+  role="presentation"
+  src="https://user-images.githubusercontent.com/2313998/205361244-8a1f527a-e1f3-45a3-ab13-5fdf82476038.png"
+/>
+</Box>
+
+### Labeling actions
+
+<Box display="flex" flexDirection="column" sx={{gap: 3}}>
+<Box display="flex" flexDirection={['column', 'column', 'column', 'column', 'row']} sx={{gap: 3}}>
+
+<Box as="p" flex={1}>
+  The row header string will be prepended to the ARIA text of action buttons. For examples, Download {`{row header}`}“,
+  “Actions:{`{row header}`}”
+</Box>
+
+<img
+  width="456"
+  role="presentation"
+  src="https://user-images.githubusercontent.com/2313998/205361247-dfea530d-9989-442b-8a50-9c66aa5d3733.png"
+/>
+</Box>
+
+<Box display="flex" flexDirection={['column', 'column', 'column', 'column', 'row']} sx={{gap: 3}}>
+
+<Box as="p" flex={1}>
+  To handle cases where the row header can be very long, we should give consumers the option to specify a shorter string
+  to identify the rows. For example, if an issue title is used as the row header, we could use the issue number instead
+  of the full issue title.
+</Box>
+
+<img
+  width="456"
+  role="presentation"
+  src="https://user-images.githubusercontent.com/2313998/205361249-763772e1-d856-479e-a632-d185c27095db.png"
+/>
+</Box>
+</Box>
+
+## Related components and patterns
+
+- [ActionBar](https://primer.style/design/components/action-bar)
```

Comment:
sorry, this is probably a silly question, but why is ActionBar in here?

Reply:
It's used for the row actions